measure_extinction/modeldata.py

```diff
@@ -145,20 +202,15 @@ def __init__(
         self.temps_min = min(self.temps)
         self.temps_max = max(self.temps)
         self.temps_width2 = (self.temps_max - self.temps_min) ** 2
-        if self.temps_width2 == 0:
-            self.temps_width2 == 1.0
```

Comment:
Is there a reason these 2 lines (and the other two sets of two lines below) are removed?  These are needed in case the width in a model parameter is zero (e.g., input a list of files w/o any variation in that parameter).